docs/upgrade-helper.md

```diff
@@ -0,0 +1,3 @@
+# Upgrade helper
+
+Point people: [@pvinis](https://github.com/pvinis)
```

Comment:
@pvinis Do you feel like fleshing this doc with explanation and instructions out?